src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
+@import 'variable.scss';
+
 body {
-  margin: 0;
+  font-family: $numberFont;
+  font-size: $numberSize;
+}
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: left;
+  box-sizing: border-box;
+  margin: 40px auto;
+  max-width: ($cellSize * 7) + ($gap * 6);
+
+  @for $i from 0 to length($monthLengths) {
```

Comment:
this loop works incorrectly. If you need any help feel free to ask at fe_chat 
<img width="1413" alt="Screenshot 2024-01-29 at 22 53 00" src="https://github.com/mate-academy/layout_calendar/assets/117587463/8eaf6918-9991-4164-8028-f8a7e178cb19">